src/components/modules/BucketViewer/index.js

```diff
@@ -61,22 +62,26 @@ const BucketViewer = (props) => {
        * @param {AWS.S3.ListObjectsV2Output} response
        */
       const filterList = async (response, path) => {
-        const filetest = new RegExp(`^${path}([\\w!\\-\\.\\*'\\(\\), ]+[/]?)$`);
+        const filetest = new RegExp(
```

Comment:
nit: I know this variable was here before but `filetest` should be in camelcase `fileTest`